.ci/tox_envs/update.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+THIS_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
`THIS_DIRECTORY="$(dirname "$0")"` should be enough here and without bashisms.

Reply:
Actually I think that was needed. Your variant gives me a relative path which I can't use twice. I've reworked the logic a bit in #4891's version of this file (https://github.com/galaxyproject/galaxy/pull/4891/commits/c3776ef0eed6c09de29b3fce6731b7795415916e#diff-95de76b45ef92f3f93bdb163d284571e) so this isn't a problem though.

Reply:
You're right, but a mix of the 2 should work:
```
THIS_DIRECTORY="$(cd "$(dirname "$0")"  > /dev/null && pwd)"
```